app/code/Magento/Customer/view/frontend/layout/customer_account_login.xml

```diff
@@ -6,6 +6,9 @@
  */
 -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <title>Customer Login</title>
```

Comment:
Please, verify that title text will be translated according to Magento locale settings

Reply:
Something like
```
<referenceBlock name="page.main.title">
            <action method="setPageTitle">
                <argument translate="true" name="title" xsi:type="string">My Dashboard</argument>
            </action>
</referenceBlock>
```
might do the trick, from [here](https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Customer/view/frontend/layout/customer_account_index.xml#L11).

Reply:
I will try to do this in this week.

Reply:
Anything on this one? I really don't want to close this PR, just let me know if you will have time in the coming weeks to work on this. Thanks!

Reply:
@miguelbalparda I tested your piece of code 

`<referenceBlock name="page.main.title">
            <action method="setPageTitle">
                <argument translate="true" name="title" xsi:type="string">My Dashboard</argument>
            </action>
</referenceBlock>`

and this work fine for page-title block in HTML but as I thought 
it has no effect for page title in `<head>` which is visible for ex. in page tab title in Chrome.

I found the small bug in Magento Framework in `\Magento\Framework\View\Page\Config\Renderer::renderTitle`

`__()` is missing there. Do we have the opportunity to fix this issue in Magento/framework?

Maybe someone has another solution for this?

Reply:
Sorry for the late reply, feel free to do a PR for magento/framework or add it to this one.